whisper_online_server.py

```diff
@@ -142,7 +148,7 @@ def format_output_transcript(self,o):
             print("%1.0f %1.0f %s" % (beg,end,o[2]),flush=True,file=sys.stderr)
             return "%1.0f %1.0f %s" % (beg,end,o[2])
         else:
-            print(o,file=sys.stderr,flush=True)
+            # No text, so no output
```

Comment:
there should be at least debug log output

Reply:
Done in 88a7f3a.